Release notes for Pipeline Control Gateway v1.1.0 (#21419)
newrelic/docs-website: src/content/docs/release-notes/pipeline-control-gateway-release-notes/pipeline-control-gateway-25-08-05.mdx

---
subject: Pipeline Control Gateway 
releaseDate: '2025-08-05'
version: 1.1.0
metaDescription: Release notes for Pipeline Control gateway 1.1.0
---

### Pipeline Control Gateway Release Notes - v1.1.0

#### Support for SQL Trace and Transaction Trace Data ####
* Functionality has been implemented to selectively drop data and attributes from SqlTrace and TransactionTrace

> #### Security Bug Fixes ####

Reviewer: ```suggestion
#### Security updates
```

* Cross-site scripting vulnerability and HTTP Proxy bypass using IPv6 Zone IDs in
 golang.org/x/net have been fixed.

> * We have upgraded to Go `1.24` and the following packages to these specified versions as part of the above fixes:

Reviewer: ```suggestion
* Upgraded to Go `1.24` and the following dependencies are updated to enhance security and incorporate recent fixes:
```

   * golang.org/x/net to v0.38.0
   * golang.org/x/sys to v0.31.0
   * golang.org/x/text to v0.23.0

> #### Support for New Functions ####

Reviewer: ```suggestion
#### Support for new functions
```

> The system now supports several new functions in NRQL drop rule queries that were previously unavailable, including

Reviewer: ```suggestion
Added support for several new functions in NRQL drop rule queries. You can now use the following functions:

- `aparse()`
- `floor()`
- `getField()`
- `hourOf()`
- `numeric()`
- `round()`
- `string()`
- `substring()`
- `weekdayOf()`
- `mod()`
- `dimensions()`

```
